modules/openshift-storage-common-terms.adoc

```diff
@@ -50,6 +50,22 @@ KMS key:: The Key Management Service (KMS) helps you achieve the required level
 
 Local volumes:: A local volume represents a mounted local storage device such as a disk, partition or directory.
 
+Nested mount points:: A nested mount point is a mount point that attempts to use a mount point created by a previous volume.
```

Comment:
```suggestion
Nested mount points:: A nested mount point is a mount point that attempts to use a mount point that a previous volume created.
```
- Minor rewrite to avoid passive voice.